deis-cluster-coreos/azuredeploy.json

```diff
@@ -0,0 +1,361 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the of the storage account for VM OS Disk"
+            }
+        },
+        "vmNamePrefix": {

```

Comment:
I see. You are using it for the DNS name of the load balancer. It does need to be a parameter, but should be called something like "DNSname" or "Loadbalancer IP DNS name"